script.js

```diff
@@ -3,60 +3,67 @@ const body = document.querySelector("body");
 body.classList.add("body");
 const searchInput = document.createElement("input");
 const cardsWrapper = document.createElement("div");
+const showDropdownSelection = document.createElement("select");
+showDropdownSelection.setAttribute("id", "show-options-value");
 const dropdownSelection = document.createElement("select");
 const options = document.createElement("option");
 const searchBarWrapper = document.createElement("section");
 const counterWrapper = document.createElement("div");
 const countH2 = document.createElement("h2");
+const showDropdownWrapper = document.createElement("form");
+const dropdownWrapper = document.createElement("form");
 
 
-/***** API *****/
-const TvShowApi = "https://api.tvmaze.com/shows/82/episodes";
-let allEpisodes;
 
+/***** ALL SHOWS API *****/
+const allShows = getAllShows();
+console.log(allShows);
+// const allShows_URL: "https://api.tvmaze.com/shows"
+
+/***** ON SETUP *****/
+window.onload = setup;
 
-/***** SET UP *****/
 function setup() {
-  fetch(TvShowApi)
-    .then(response => {
+  // getEpisodeData();
+  pageTitle();
+  createTvMazeLink();
+  searchAreaWrapper();
+  allShowsDropdown();
+  // createEpisodeDropdown();
+  searchBar();
+  showsAndEpisodeCounter();
+  showCount(allShows, allShows);
+  createCardsWrapper();
+  showCards(allShows);
+}
+
+/***** EPISODES API *****/
+let selectedShow;
+// let episodes_API = `https://api.tvmaze.com/shows/22/episodes`;
+let allEpisodes = [];
+console.log(allEpisodes); // << undefined
+
+function getEpisodeData(showId) {
+  const episodes_API = `https://api.tvmaze.com/shows/${showId}/episodes`;
+  console.log(episodes_API);
+  return fetch(episodes_API)
+    .then((response) => {
       if (response.ok) {
         return response.json();
       }
       throw `${response.status} ${response.statusText}`;
     })
-    .then(data => {
+    .then((data) => {
       allEpisodes = data;
-      console.log(data);
-      projectTitle();
-      showTitle();
-      createTvMazeLink();
-      createSearchArea();
-      countH2.textContent = `${allEpisodes.length} / ${allEpisodes.length} Episodes`;
-      createCardsWrapper();
-      displayEpisodeCards(allEpisodes);
+      console.log(allEpisodes);
     })
-    .catch(error => {
+    .catch((error) => {
       console.log("An Error Occurred:", error);
     });
 }
 
-window.onload = setup;
-
-
-/***** PROJECT TITLE *****/
-
-function projectTitle() {
-  const titleWrap = document.createElement("div");
-  titleWrap.classList.add("title-wrapper");
-  const header = document.createElement("h1");
-  header.innerText = "TV DOM Project";
-  body.appendChild(titleWrap);
-  titleWrap.appendChild(header);
-}
-
-/***** SHOW TITLE *****/
-
-function showTitle() {
+/***** PAGE TITLE *****/
+function pageTitle() {
   //SHOW TITLE WRAPPER
   const showTitleWrapper = document.createElement("div");
```

Comment:
Because you have only one element (showTitle -h1) inside this div (showTitleWrapper), I don't think it was necessary to create a parent div. You could easily manipulate the h1 without the div. It is not something wrong but it's more work you had to put in.